_layouts/post.html

```diff
@@ -17,14 +17,14 @@ <h1 class="page-title">{{page.title}}</h1>
 
             {% if page.qa_video %}
                 <div class="smallhead">
-                This week's Instructor Q&A session has been completed.
+                This week's Instructor Q&A session has ended.
                 Please see the bottom of this page for the recording of the session.
                 </div>
 
                 <div class="bigspacer"></div>
 
                 <div class="smallhead">
-                <b><a class="off" href="{{ page.qa_doc }}">{{page.title}} Q&A Doc</a></b>: This document is frozen, but contains answers to the presubmitted Q&A.
+                <b><a class="off" href="{{ page.qa_doc }}">{{page.title}} Q&A Doc</a></b>: Click here for answers to the Q&A Doc.
```

Comment:
Doesn't "Click here" imply that the hyperlink will be on the word "here"?

Reply:
Sorry about that. I suppose so, yes. I'll update to include a link for "here"